UniswapV3Adapter/contracts/UniswapV3Adapter.sol

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Please order functions layout by type
Refer https://docs.soliditylang.org/en/v0.8.17/style-guide.html#order-of-layout